functional_tests/features/bundle_page.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Publishing Officer can draft, edit, and publish bundles
+# Viewer
+# Publishing Admin
+# Publishing Officer
+    Background:
```

Comment:
The `Background` block gets executed for all of your scenarios. This is why your scenarios are failing, because you are already logged in as a "Publishing Officer" when you try to log in as other users.

I think what you meant to do here is make this a scenario:
```suggestion
    Scenario: A Publishing Officer can add Bundles
```